.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*]
```

Comment:
of course, but there is one for Brooklyn. Whilst this isn't a part of the open source project they are linked and used by the community. I wonder if it is worth discussing with the Brooklyn mailing list about using editorconfig or similar as a standard